azure-pipelines-full-tests.yml

```diff
@@ -16,65 +16,8 @@ variables:
 - template: ${{ variables.Pipeline.Workspace }}/.azure-pipelines/templates/variables.yml
 
 jobs:
-- job: AutomationTest20200901
-  displayName: Automation Test (Profile 2020-09-01)
```

Comment:
I think these tests should be kept for LTS which still supports Azure Stack.

Reply:
You're right, I removed them because the tests are fails and they has been removed in dev branch. Didn't realize they're part of Azure Stack.